src/main.c

```diff
@@ -147,12 +157,23 @@ __attribute__((section(".boot"))) int main(int arg0) {
                 if (args[0] != ETH_PLUGIN_CHECK_PRESENCE) {
                     dispatch_plugin_calls(args[0], (void *) args[1]);
                 }
-
-                // Call `os_lib_end`, go back to the ethereum app.
-                os_lib_end();
             }
         }
+        CATCH_OTHER(e) {
+            switch (e) {
+                // These exceptions are only generated on handle_query_contract_ui()
+                case 0x6502:
```

Comment:
Do we have a defined name for this one ?

Reply:
I did not find it...
https://github.com/LedgerHQ/ethereum-plugin-sdk/blob/0de74c6382f876f89f82ef0eef90408fd94888a3/include/eth_internals.c#L228